client/src/routes/SessionModal/SessionModal.tsx

```diff
@@ -138,9 +188,18 @@ const SessionModal = () => {
       'Session Type': session.type,
       'Session Start Time': session.startTime,
     });
-  };
+  }, [
+    exercise?.name,
```

Comment:
Same as above, thinks it's ok to depend on `session` and `exercise` here

Reply:
Yeah exercise it actually makes no sense to not use the whole thing as a dependency since useExerciseById memoizes the whole content by id already